javascript/elements/updates_for_element.js

```diff
@@ -32,6 +32,9 @@ export default class UpdatesForElement extends SubscribingElement {
     super()
     const shadowRoot = this.attachShadow({ mode: 'open' })
     shadowRoot.innerHTML = template
+
+    this.triggerElementLog = []
+    this.targetElementLog = []
```

Comment:
Do we need to clear these logs or perhaps have a log stack that we move them to when `cable-ready:after-update` is dispatched?

Reply:
🤔

You are probably correct, though in my tests it didn't really matter because they are cleared with every browser refresh.

I think the easiest would be to limit them to a small number (10?) and do a kind of FIFO thing so they get cleared automatically?